src/Modal/Modal.jsx

```diff
@@ -86,7 +86,7 @@ export const Modal = ({
       {...props}
       bg={backDrop}
       animated={animated}
-      onClick={modalClickHandler}>
+      onMouseDown={modalClickHandler}>
```

Comment:
1. I was expecting the modal to close on mouse up ... 
2. why is this fixing the bug ? what's the difference ?